plugins/convert_README_to_header.sh

```diff
@@ -30,8 +32,9 @@ outfilename=`basename $outfile`;
 # See http://stackoverflow.com/a/16576291
 # On Mac OS's sed, \n is not recognized as a newline character, but
 # \[actual newline] works
-desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\"/\\\"/g' -e 's/\n/\\\\n"\\
-"/g' "$path/README.md"`;
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\\\/\\\\\\\/g' -e 's/\"/\\\"/g' \
```

Comment:
I assume `s/\\\/\\\\\\\/g` is trying to escape a backslash, or an escaped backslash? I don't think there are currently enough backslashes, you've got three replaced with seven. I think the shell will eat half of them, leaving `s/\/\\\/g`, which sed would interpret as an escaped / followed by a \ and escaped /, but with no substitution operators. I think you actually want four and eight, which boils down to two/four and therefore one/two. See the quote escaping for example.

Reply:
We're escaping backslash:
- first for bash so sed 'sees' only 2
- then for sed since backslash has significance there too

Reply:
>     * first for bash so sed 'sees' only 2
We'll need one per sed backslash (escaping or real), rather than one in total.

>     * then for sed since backslash has significance there too
Likewise here.

I believe you're trying to logically replace `\` with `\\`?

Reply:
Yes because backslash also has significance in C into which the readme is ultimately converted.